cumulusci/robotframework/Salesforce.py

```diff
@@ -258,72 +198,69 @@ def go_to_setup_object_manager(self):
-        self._call_selenium("_header_field_value_should_not_exist", True, locator)
+        self.selenium.page_should_not_contain_element(locator)
 
+    @selenium_retry
     def header_field_should_have_link(self, label):
         """ Validates that a field in the record header has a link as its value """
         locator = lex_locators["record"]["header"]["field_value_link"].format(label)
-        self._call_selenium("_header_field_value_should_exist", True, locator)
+        self.selenium.page_should_contain_element(locator)
 
+    @selenium_retry
     def header_field_should_not_have_link(self, label):
         """ Validates that a field in the record header does not have a link as its value """
         locator = lex_locators["record"]["header"]["field_value_link"].format(label)
-        self._call_selenium("_header_field_value_should_not_exist", True, locator)
+        self.selenium.page_should_not_contain_element(locator)
 
+    @selenium_retry
     def header_field_should_be_checked(self, label):
         """ Validates that a checkbox field in the record header is checked """
         locator = lex_locators["record"]["header"]["field_value_checked"].format(label)
-        self._call_selenium("_header_field_value_should_exist", True, locator)
+        self.selenium.page_should_contain_element(locator)
 
+    @selenium_retry
     def header_field_should_be_unchecked(self, label):
         """ Validates that a checkbox field in the record header is unchecked """
         locator = lex_locators["record"]["header"]["field_value_unchecked"].format(
             label
         )
-        self._call_selenium("_header_field_value_should_exist", True, locator)
-
-    def _header_field_value_should_exist(self, locator):
         self.selenium.page_should_contain_element(locator)
 
-    def _header_field_value_should_not_exist(self, locator):
-        self.selenium.page_should_not_contain_element(locator)
-
+    @selenium_retry
     def open_app_launcher(self):
-        """ EXPERIMENTAL!!! """
+        """ Opens the Saleforce App Launcher """
         locator = lex_locators["app_launcher"]["button"]
-        self._call_selenium("_open_app_launcher", True, locator)
-
-    def _open_app_launcher(self, locator):
-        self.builtin.log("Hovering over App Launcher button")
-        self.selenium.mouse_over(locator)
+        #self.builtin.log("Hovering over App Launcher button")
+        #self.selenium.mouse_over(locator)
+        #time.sleep(1)
         self.builtin.log("Clicking App Launcher button")
-        self.selenium.get_webelement(locator).click()
-        self.builtin.log("Waiting for modal to open")
+        self.selenium.click_button(locator)
         self.wait_until_modal_is_open()
 
+    @selenium_retry
     def populate_field(self, name, value):
         locator = lex_locators["object"]["field"].format(name)
-        self._call_selenium("_populate_field", True, locator, value)
+        self._populate_field(locator, value)
```

Comment:
This method doesn't call selenium directly; should the decorator go on _populate_field instead?